usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -255,9 +255,20 @@ def keyword_parse(keyword):
                 queryset = queryset.filter(cfda_number__in=in_query)
 
         elif key == "naics_codes":
-            if value:
-                regex = f"^({'|'.join([str(elem) for elem in value])}).*"
-                queryset = queryset.filter(naics_code__regex=regex)
+            if isinstance(value, list):
+                require = value
+            elif isinstance(value, dict):
+                require = value.get("require") or []
+                if value.get("exclude"):
+                    raise NotImplementedException(
+                        "NOT IMPLEMENTED: postgres endpoint does not currently support excluded naics!"
+                    )
+
+            else:
+                raise InvalidParameterException("naics_codes must be an array or object")
+
+            regex = f"^({'|'.join([str(elem) for elem in require])}).*"
+            queryset = queryset.exclude(naics_code__regex=regex)
```

Comment:
If I'm understanding this right. It's excluding the elements we're requiring? Is this a typo or am I missing something?

Reply:
that looks like a huge typo